iib/workers/config.py

```diff
@@ -44,6 +44,7 @@ class Config(object):
     iib_image_push_template: str = '{registry}/iib-build:{request_id}'
     iib_index_image_output_registry: Optional[str] = None
     iib_log_level: str = 'INFO'
+    iib_deprecate_bundles_limit = 200
```

Comment:
How many  bundles are usually deprecated?
How many bundles needs to be deprecated for community?

Reply:
For the community, I counted ~ 3200 in the logs. For the Prod v4.19, there were 45 deprecations.

Reply:
In most cases, we will not even come close to 200. The community index is very different from the other ones. In our case and setup, it would be safe to increase this limit to something like 500 (maybe even 1000), but the 200 will work with a smaller stack size, so it should prevent this problem in the future.

Reply:
The code looks OK. The question is if `opm registry deprecatetruncate` command has significant "loading" time (for example builds a cache). Because if we keep limit 200 then we increase the processing time by the "loading opm time" 16x. 

Example: I know `opm render` takes more then 1 minute. So it would be extra 16 minutes and I think it is significant amount. Maybe `opm registry deprecatetruncate`  is fast and we can merge this. 

Could you try it at post here what have you learned?

Reply:
The `opm deprecatetruncate` took at most ~4 seconds with around ~700 bundles for the deprecation.